mlrun/db/base.py

```diff
@@ -726,5 +726,11 @@ def enable_model_monitoring(
         project: str,
         base_period: int = 10,
         image: str = "mlrun/mlrun",
-    ):
-        pass
+        deploy_histogram_data_drift_app: bool = True,
+    ) -> None:
+        raise NotImplementedError
```

Comment:
All the other unimplemented methods in this class are passed not raising anything, I think better to stick with the same behavior.

Reply:
agree should be consistent, we need to decide if should raise an expception or to pass.

Reply:
Addressed in #5418 as well as the other comments.